### DISCUSSION TITLE
AuthorizationCodeRefreshResponse has no refresh token property

### DISCUSSION THREAD

**Discussion by ThresherXG** [JohnnyCrazy/SpotifyAPI-NET#666] [Ideas]
From Spotify's api I see when a token is requested via refresh token a new token and refresh token are returned. The model for the refresh response does not have a refresh token property. Can this be added?

Sorry if I missed it, or I am using this incorrectly.

Thanks!

**Comment by JohnnyCrazy** [comment]
Good catch, fixed in master and soon released as hotfix! 👍🏼 

Thank you

**Reply by ThresherXG** [reply]
Awesome, thank you!

**Comment by guilhermeloyola** [comment]
The property is returning null. Is any chance that im doing wrong or maybe its not returning?
